pulsarlib/messaging.go

```diff
@@ -361,24 +379,93 @@ func CreateConsumer(tenantID string, namespace string, topics []string, subscrip
 	for _, tp := range topics {
 		topicArr = append(topicArr, fmt.Sprintf("persistent://%s/%s/%s", tenantID, namespace, tp))
 	}
-	c, err := msging.client.Subscribe(pulsar.ConsumerOptions{
+	consumerOptions := pulsar.ConsumerOptions{
```

Comment:
Can we put validation on opts passed. Subscription name can be empty. Does pulsar raise an error in that case

Reply:
yes, it raise an error if subscription is empty